src/composables/useApiGetMerchandise.ts

```diff
@@ -111,8 +126,205 @@ export const useApiGetMerchandise = (
     }
   };
 
+  const merchandiseGroupedByName = computed(() => {
+    return merchandise.value.reduce(
+      (acc, item) => {
```

Comment:
Same as my comments bellow.

Reply:
Renamed to groups + refactored the reduce function to standalone `groupMerchandiseByKey` function (based on your other comment).